pkg/issuer/venafi/client/venaficlient.go

```diff
@@ -187,6 +190,84 @@ func configForIssuer(iss cmapi.GenericIssuer, secretsLister corelisters.SecretLi
 	return nil, fmt.Errorf("neither Venafi Cloud or TPP configuration found")
 }
 
+// httpClientForVcertTPP creates an HTTP client and customises it to allow client TLS renegotiation.
+//
+// Here's why:
+//
+//  1. The TPP API server served by Microsoft Windows Server + IIS.
```

Comment:
nitpick (non-blocking): typo / missing word!

```suggestion
//  1. The TPP API server is served by Microsoft Windows Server + IIS.
```

Reply:
Done.